applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -611,6 +670,13 @@ The application supports the following configurations:
      - Release version of the application.
 
        Enables only the necessary application functionalities to optimize its performance.
+   * - Matter bridge over Wi-Fi
+     - :file:`prj_nrf70ek.conf`
+     - ``nrf70ek``
+     - nRF5340 DK with the nRF7002 EK shield attached
+     - Debug version of the application with Matter over Wi-Fi enabled.
+
+       Enables Matter-Bridge to work with Wi-Fi.
```

Comment:
```suggestion
       Enables the Matter Bridge to work with Wi-Fi on nRF5340 DK.
```

Reply:
```suggestion
       Enables Matter bridge to work over Wi-Fi.
```
We don't need to specify the device here either, since there's a specific entry for supported devices in the table. We also don't mention the devices for the similar network switching in the Door lock sample.